test/sim/abilities/myceliummight.js

```diff
@@ -19,4 +19,22 @@ describe("Mycelium Might", function () {
 		battle.makeChoices();
 		assert.false.fullHP(battle.p2.active[0]);
 	});
+	it("should never trigger your own quick claw if using a status move", function () {
+		battle = common.createBattle({seed: [3, 1, 1, 1]}, [[
```

Comment:
I think you can use {forceRandomChance: true} here to avoid a hardcoded seed, same for the second test.

Reply:
Didn't know that was a thing, cool. Implemented